### PR TITLE
testcase: add test case for issue #16100

### PR DESCRIPTION
**Pull request by ricardoquesada**
Fixes github issue #16100

### CODE REVIEW

**Comment by ricardoquesada** [comment]
[ci rebuild]

File: tools/cocos2d-console [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 2ceba42b89945229ac22d09c0864c86b113f3f17
```
Comment (minggo) [inline tools/cocos2d-console:1]: I think there is not need to change console submodule here.

Comment (ricardoquesada) [inline tools/cocos2d-console:1]: oops. thanks. I'll fix it.

Comment (ricardoquesada) [inline tools/cocos2d-console:1]: Apparently it was fixed here: https://github.com/cocos2d/cocos2d-x/commit/3e33b7a1cad14dab8bd896c270e319e2d12a78e3
thanks.

**Pull request merged**